src/components/Comments.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../data";
+
+const Comments = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [IsLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getComments(article_id).then((commentsAPI) => {
+      setComments(commentsAPI);
+      setIsLoading(false);
```

Comment:
This is nice, you've sert the loading but then not actually used it anywhere. 

- [ ] Could we get some feedback the comments are loading?